From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sun, 27 Mar 2011 15:19:15 +0200
Subject: [PATCH] don't link against libelf

libelf is not used. It's just some broken configure check.
Patch from gentoo.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
[osterlad: Update locations for version 4.5.0]
Signed-off-by: Denis Osterland <Denis.Osterland@diehl.com>
---
 configure.ac | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index ffe2e372426c..d88af75af665 100644
--- a/configure.ac
+++ b/configure.ac
@@ -202,14 +202,6 @@ AC_EGREP_CPP(yes,
 #endif
 ], LIBS="$LIBS -lsocket -linet";seqptx=1)
 
-oldlibs="$LIBS"
-LIBS="$LIBS -lelf"
-AC_CHECKING(SVR4)
-AC_TRY_LINK([#include <utmpx.h>
-],,
-[AC_CHECK_HEADER(dwarf.h, AC_DEFINE(SVR4) AC_DEFINE(BUGGYGETLOGIN),
-[AC_CHECK_HEADER(elf.h, AC_DEFINE(SVR4) AC_DEFINE(BUGGYGETLOGIN))])]
-,LIBS="$oldlibs")
 AC_CHECK_HEADERS([stropts.h string.h strings.h])
 
 AC_CHECKING(for Solaris 2.x)
